runtime/gc_glue_java/MetronomeDelegate.cpp

```diff
@@ -558,7 +558,9 @@ MM_MetronomeDelegate::addDyingClassesToList(MM_EnvironmentRealtime *env, J9Class
 								(UDATA) J9UTF8_LENGTH(J9ROMCLASS_CLASSNAME(clazz->romClass)),
 								J9UTF8_DATA(J9ROMCLASS_CLASSNAME(clazz->romClass)));
 					TRIGGER_J9HOOK_VM_CLASS_UNLOAD(_javaVM->hookInterface, vmThread, clazz);
-						
+
+					_javaVM->internalVMFunctions->freeMapCaches(classLoader);
+
```

Comment:
Similarly with previous comment if this function intended to be call once per class loader it should be added to `MM_MetronomeDelegate::processDyingClasses()` at line 494.